person_capture/gui_app.py

```
@@ -3844,11 +3986,12 @@ def _pipe_like(obj) -> bool:
                     self._hdr_preview_latest = None
-                    try:
-                        pos = int(cap.get(cv2.CAP_PROP_POS_FRAMES) or 0)
-                        self._hdr_preview_seek(pos)
-                    except Exception:
-                        pass
+                    if (not prescan_will_run) or allow_prescan_hdr_preview:
+                        try:
+                            pos = int(cap.get(cv2.CAP_PROP_POS_FRAMES) or 0)
+                            self._hdr_preview_seek(pos)
+                        except Exception:
+                            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)